knexfile.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
```

Comment:
What's the reason for disabling eslint here? Is it the same SQL naming convention reason you mention below?

Reply:
It complains when I use `use strict` because our eslint config is setup for modules and in modules you don't need it, but I'm pretty sure one should use `use strict` in normal node.js scripts